net/fcgiwrap/Makefile

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2017 Alf Brand <git.alf@alftand.de>
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=fcgiwrap
+PKG_VERSION:=1.1.0
+PKG_RELEASE:=1
+
+PKG_SORUCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/gnosek/fcgiwrap.git
```

Comment:
Hmm... This is a bit sneaky but can't we use the tarball from Gentoo's distfiles instead of pulling git ourselves?
https://github.com/lede-project/source/pull/1133/files
fcgiwrap-1.1.0.tar.gz